src/Discovery/DiscoveryConfig.d.ts

```diff
@@ -0,0 +1,92 @@
+/* eslint-disable camelcase */
```

Comment:
is there a reason to not use camelcase for these configuration fields?

Reply:
Great point, no there isn't, plus it's different from the portal config. I'll update the config to use camelcase!